Program.cs

```diff
@@ -548,6 +556,68 @@ static void RandomTaco()
 
     #endregion
 
+    #region Covid-19
+        
+      static void Covid19()
+      {
+        Console.WriteLine("");
+        Console.WriteLine("Enter the country name to get the information. If you want to see the global information, type \"Global\". For the list of all countries type \"List\": ");
+        string country = Console.ReadLine();
```

Comment:
Call this userInput or infoType instead of "country" - since its only available options are "Global" or "List"

Reply:
The countries are available options too